Markdown editor with preview and editor tabs

HarmvZ opened the pull request:
https://user-images.githubusercontent.com/2001094/128737744-7466182a-a223-42da-a281-0b5f5dc9f468.mp4

jmsmkn reviewed (changes requested):
I don't think that this will work with multiple widgets in the same form. Can you update all ids, hrefs, and arias to include `{{ widget.attrs.id }}`?

jmsmkn commented:
Also can you check how this is in the admin?

HarmvZ commented:
> I don't think that this will work with multiple widgets in the same form. Can you update all ids, hrefs, and arias to include {{ widget.attrs.id }}?

Good catch, that did not work indeed. Fixed it now.

The admin also did not work as it should because it was missing the vendor bootstrap js files. Added those also. The spacing/indentation is a bit strange, but I think that's not really important. It looks like this:

![image](https://user-images.githubusercontent.com/2001094/128827802-6a2134b8-9e9b-4fc0-b58e-c1244e09add4.png)
